### ISSUE TITLE
Error during Droopler installation 

### ISSUE THREAD

**Issue by valica** [droptica/droopler_project#137]
Hello,

I've just tried to use the install procedure available on git.
I've started with 
composer create-project droptica/droopler-project  droopler8310 "^8.3.1"
(this is for you to know the profile version)

Then I followed usual steps (and that's not the first time I install Droopler).
Everything worked well till the site install.
At site install, during the "Install additional components" step, I've got an error which is:
n AJAX HTTP error occurred.
HTTP Result Code: 200
Debugging information follows.
Path: /droopler/core/install.php?rewrite=ok&profile=droopler&langcode=en&id=2&op=do_nojs&op=do
StatusText: OK
ResponseText: Drupal\Core\Database\InvalidQueryException: Query condition 'nfd.nid IN ()' cannot be empty. in Drupal\Core\Database\Query\Condition->condition() (line 117 of C:\Data\webdev\drupalProjects\droopler8310\web\core\lib\Drupal\Core\Database\Query\Condition.php).

I've tried to make the installation again and choose less features to install, keeping only d_produts and the documentation but same error at the end.
The installation may continue then but the site is not usable.

Is a different profile version that fix this issue?

Kind regards,

Vlad

**Comment by grzegorz-pietrzak-droptica** [comment]
Hello, perhaps it is related to https://www.drupal.org/project/droopler/issues/3324922

**Comment by valica** [comment]
Hello,
It might be, I've just checked and selecting all additional modules (so including d_blog) was leading to a 'no error" install.
But there is still a problem once the installation is done.
The d_p_banner paragraphs do not show the background image. Looking to the problem, I saw that the imege link is incorrectly built. My installation is under a "droopler" virtual host on Apache webserver. This means my root website is at this URL: http://localhost/droopler/
When the d_p_banner is loading, the image link is http://localhost/profiles/contrib/droopler/modules/custom/d_demo/assets/d_demo_tile_plane.jpg  which is wrong because "droopler" is missing after localhost. 
This is also the same for other images and also for 	/libraries/lazysizes/lazysizes.min.js

This is also the same for d_p_carousel. 
However other images gets loaded correctly, using the good path like the ones in group of text blocks:
/droopler/sites/default/files/styles/small_box_image/public/demo/d_product_init_centurion-3000-1.jpg
and
/droopler/profiles/contrib/droopler/modules/custom/d_demo/assets/demo-logo.svg

I performed clear cache and same problem.
So why the root folder 'droopler' is missing ?

If you have any hint, it will help me because I'm stuck.
Kind regards,
Vlad

**Comment by grzegorz-pietrzak-droptica** [comment]
I guess nobody tested Droopler in this way so far - installing in a subdirectory is not a common case. I would rather suggest configuring your server to use a domain like `droopler.localhost`. We'll try to fix it in the new version.

**Comment by valica** [comment]
In fact I have several sites hosted and served by a same Apache or NGINX server. That's why each of them are located in a dedicated folder. If you have a hint concerning the origin of this problem, I'm interested t eget it because I can temporarly fix on my own and everything would be OK once you deliver an update.

In http://localhost/droopler/
droopler is just an alias pointing to a virtual host directory.

Thank you for a hint waiting for the official fix.
Kind regards

**Comment by grzegorz-pietrzak-droptica** [comment]
Most probably, our demo building script is not taking this additional directory into account. For now the only way to fix it is to do a search&replace in the DB, or to patch `d_demo`/`d_content_init` modules before the install.

**Comment by grzegorz-pietrzak-droptica** [comment]
The ticket was added to https://www.drupal.org/project/droopler/issues/3341290

**Comment by kwiktorowicz-droptica** [comment]
I installed the Droopler on localhost via
"composer create-project droptica/droopler-project . "^8.3.1""
in the subfolder http://testowastrona.local/test3/web/ - installation went without any problems.

I went through the entire installation process, with all the additional modules. I had no errors either during or after installation.

**Comment by kwiktorowicz-droptica** [comment]
I cannot reproduce the problem.

I installed the Droopler on localhost via
"composer create-project droptica/droopler-project . "^8.3.1""
in the subfolder http://testowastrona.local/test3/web/ - installation went without any problems.

I went through the entire installation process, with all the additional modules. I had no errors either during or after installation.

**Comment by grzegorzbartman** [comment]
We are moving issue management to Drupal.org. If this problem still exists, please create a new issue here: https://www.drupal.org/node/add/project-issue/droopler
